extensions/ql-vscode/test/unit-tests/common/github-url-identifier-helper.test.ts

```diff
@@ -35,6 +34,7 @@ describe("github url identifier helper", () => {
 
     it("should handle valid urls", () => {
       expect(getNwoFromGitHubUrl("https://github.com/foo/bar")).toBe("foo/bar");
+      expect(getNwoFromGitHubUrl("http://github.com/foo/bar")).toBe("foo/bar");
```

Comment:
Could we also add a check for just `"github.com/foo/bar"`?

Reply:
good idea!